### PR TITLE
Create Dashboard Activity layout

### PR DESCRIPTION
**Pull request by josemontesdeoca**
Dashboard Activity Layout following our current wireframes https://www.flinto.com/p/a0de3ed3

/cc @fcduarte @amitbharadwaj

### CODE REVIEW

File: res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="white">#FFFFFF</color>
```
Comment (fcduarte) [inline res/values/colors.xml:4]: We could use the system values for white and gray.

File: src/org/missionassetfund/apps/android/activities/MainActivity.java [hunk 2]
```diff
@@ -2,15 +2,54 @@
...
+    
+    @SuppressWarnings("rawtypes")
+    private void showFragment(Class activeFragmentClass) {
+        Class[] fragmentClasses = new Class[] {
+                DashboardFragment.class
+        };
+        FragmentManager mgr = getSupportFragmentManager();
+        FragmentTransaction transaction = mgr.beginTransaction();
+        try {
+            for (Class klass : fragmentClasses) {
```
Comment (fcduarte) [inline src/org/missionassetfund/apps/android/activities/MainActivity.java:35]: Do you prefer `klass` over `clazz` ? :)

**Comment by fcduarte** [comment]
Reviewed.. Just a few comments to discuss.

File: src/org/missionassetfund/apps/android/activities/MainActivity.java (hunk 2 above)
Comment (josemontesdeoca) [inline src/org/missionassetfund/apps/android/activities/MainActivity.java:35]: I'm just lazy and used the pattern shown [here](https://github.com/thecodepath/tumblrsnap/blob/master/src/com/codepath/apps/tumblrsnap/activities/MainActivity.java#L76)

Comment (fcduarte) [inline src/org/missionassetfund/apps/android/activities/MainActivity.java:35]: np!

2014-07-07 13:50 GMT-07:00 Jose Montes de Oca notifications@github.com:

File: res/values/colors.xml (hunk 1 above)
Comment (josemontesdeoca) [inline res/values/colors.xml:4]: @fcduarte we could but its best to trying to decouple from the SDK. Let me know if you disagree here.

Comment (fcduarte) [inline res/values/colors.xml:4]: Ok let's use this pattern so please update line 19 on `res/drawable/bg_card.xml`

Comment (josemontesdeoca) [inline res/values/colors.xml:4]: @fcduarte bg_card.xml is now using our define colors. Let me know if anything else is needed.

Comment (fcduarte) [inline res/values/colors.xml:4]: image

**Pull request merged**